Lib/fontTools/feaLib/parser.py

```diff
@@ -764,8 +755,21 @@ def parse_nameid_(self):
         elif platformID == 3 and platEncID == 1:
             string = self.unescape_windows_name_string(string)
 
-        return ast.NameRecord(location, nameID, platformID,
-                              platEncID, langID, string)
+        return platformID, platEncID, langID, string
+
+    def parse_nameid_(self):
+        assert self.cur_token_ == "nameid", self.cur_token_
+        location, nameID = self.cur_token_location_, self.expect_number_()
+        if nameID > 255:
+            raise FeatureLibError("Expected name id < 255",
+                                  self.cur_token_location_)
+        if 1 <= nameID <= 6:
+            # TODO: issue a warning
```

Comment:
Agreed, I even tried to use the logging module here the first time I wrote this code, I just wasn’t sure what the format of the warning should be, so I left it in the hope that someone else will take care of it :)